chia/protocols/full_node_protocol.py

```diff
@@ -203,3 +203,12 @@ class RequestPeers(Streamable):
 @dataclass(frozen=True)
 class RespondPeers(Streamable):
     peer_list: List[TimestampedPeerInfo]
+
+
+@streamable
+@dataclass(frozen=True)
+class WrappedCompressed(Streamable):
+    # The inner (wrapped) type's ProtocolMessageTypes enum
+    inner_type: uint8
```

Comment:
this type is not used anywhere, it makes the protocol less robust as changing this type won't highlight any of the places that need to agree on this protocol. nor will it actually change the protocol.

Reply:
uint8 is used because the field in Message has that type.

Reply:
Arvid, I believe this would come down to `Streamable` not supporting enumerations.  I had suggested previously that we make it so it does since this would avoid hazards around hand-coded type selection when processing the messages, but there were other concerns.